docs/control/administering/debugging.rst

```diff
@@ -9,8 +9,11 @@ Debugging
 Logging
 -------
 
-Flocker processes use `eliot`_ for logging.
-These logs can be rendered as an ASCII tree using `eliottree`_.
+Flocker processes use the `Eliot`_ framework for logging.
+Eliot structures logs as a tree of actions, which means given an error you can see what Flocker actions caused the errors by finding the other messages in the tree.
+The tree of actions can also span processes; thus you can trace API calls from within the Docker plugin and see the effects in the control service logs.
+Messages can be rendered into a human-readable tree using the `eliot-tree`_ command-line tool, which is pre-installed with Flocker.
```

Comment:
We're pretty inconsistent when we use "command-line" and "command line" throughout the docs - I think we should be using it without the dash, but happy to pick this up myself in FLOC 3237.